### DISCUSSION TITLE
How to stop any unknown person from following me?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#48233] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi.

I would like to prevent people from following me if I don't know. There are users I don't know following me, how can I avoid this? How to stop any unknown person from following me?

**Comment by radar-blip** [comment, marked as answer]
You cant stop people from following you, but you can block people from viewing your activities.
Read here for more info on that: https://docs.github.com/en/communities/maintaining-your-safety-on-github/blocking-a-user-from-your-personal-account
